website/pages/docs/configuration/telemetry.mdx

```diff
@@ -37,6 +37,21 @@ The following options are available on all telemetry configurations.
 - `enable_hostname_label` `(bool: false)` - Specifies if all metric values should
   contain the `host` label with the local hostname. It is recommended to enable
   `disable_hostname` if this option is used.
+- `lease_metrics_epsilon` `(string: "1hr")` - Specifies the the size of the bucket used to measure future 
+  lease expiration. For example, for the default value of 1 hour, the `vault.expire.leases.by_expiration`
+  metric will aggregate the total number of expiring leases for 1 hour buckets, starting from the current time.
+  Note that leases are put into buckets by rounding. For example, if `lease_metrics_epsilon` is set to 1hr and 
+  lease A expires 25 minutes from now, and lease B expires 35 minutes from now, then lease A will be in the first
+  bucket, which corresponds to 0-30 minutes, and lease B will be in the second bucket, which corresponds to 31-90
+  minutes.
+- `num_lease_metrics_buckets` `(int: 168)` - The number of expiry buckets for leases. For the default value, for
+  example, 168 value labels for the `vault.expire.leases.by_expiration` metric will be reported, where each value 
+  each bucket is separated in time by the `lease_metrics_epsilon` parameter. For the default 1 hour value of 
+  `lease_metrics_epsilon` and the default value of `num_lease_metrics_buckets`, `vault.expire.leases.by_expiration`
+  will report the total number of leases expiring within each hour from the current time to one week from the current time.
```

Comment:
I feel like "For the default value, for example, 168 value labels..." is awkward.

 ```suggestion
- `num_lease_metrics_buckets` `(int: 168)` - Controls the number of buckets reported by the `vault.expire.leases.by_expiration` metric, where each bucket is separated in time by the `lease_metrics_epsilon` parameter. For the default 1 hour value of  `lease_metrics_epsilon` and the default value of 
`num_lease_metrics_buckets`, `vault.expire.leases.by_expiration` will report the total number of leases 
expiring within each hour from the current time to one week from the current time.
```